Soleras3/react-challenge-project-jan-2023
# Feature: User Display in Nav

## Changes
_List Changes Introduced by this PR_
1. Add email display in navigation bar.
2. Fix navigation padding issue introduced in #9. (This could be split into a separate issue.)

Depends on #9 - keeps feature dev on up-to-date dependencies.

## Purpose
Indicates that the user is logged in. https://github.com/Shift3/react-challenge-project-jan-2023/issues/10

## Approach
The simplest solution is sometimes best to meet user requirements.

A more complicated implementation could re-design the navigation bar for different aesthetics. Example: Navigation buttons move to left 50%; right-align user name. Nav bars often highlight the user because you can interact with that element to do various actions like view profile or log out.

## Pre-Testing TODOs
- Requires #5

## Testing Steps
- Log in with a user email and password.
- Observe the user email displayed in the navigation bar inline with "Log Out".

## Learning
Spent some time looking at example navigation bars for fun. Consider the one on this very github page!
Closes https://github.com/Shift3/react-challenge-project-jan-2023/issues/10
